server/controllers/nextPayment.controllers.js

```diff
@@ -30,7 +30,7 @@ const NextPaymentController = {
 
         res.status(httpStatus.OK).json({
           success: true,
-          message: 'Reminders sent',
+          message: `${reminders.length} ${reminders.length > 1 ? 'reminders' : 'reminder'} sent`,
```

Comment:
Zero is always followed by plural nouns so you might want to change the logic

https://english.stackexchange.com/questions/38293/why-is-zero-followed-by-a-plural-noun

```suggestion
message: `${reminders.length} ${reminders.length === 1 ? 'reminder' : 'reminders'} sent`,
```